meson.build

```diff
@@ -115,14 +116,20 @@ else
   dxvk_wsi = get_option('dxvk_native_wsi')
   
   if dxvk_wsi == 'sdl2'
-    lib_sdl2 = cpp.find_library('SDL2')
+    sdl2_dep = dependency('sdl2')
     compiler_args += ['-DDXVK_WSI_SDL2']
   elif dxvk_wsi == 'glfw'
-    lib_glfw = cpp.find_library('glfw')
+    glfw_dep = dependency('glfw3')
     compiler_args += ['-DDXVK_WSI_GLFW']
   endif
-  
-  dxvk_name_prefix = 'libdxvk_'
+
+  if dxvk_wsi == 'sdl2'
+    dxvk_name_prefix = 'dxvk_'
+    dxvk_pkg_prefix = 'dxvk-'
+  else
+    dxvk_name_prefix = 'dxvk_@0@_'.format(dxvk_wsi)
+    dxvk_pkg_prefix = 'dxvk-@0@-'.format(dxvk_wsi)
+  endif
```

Comment:
It's a bit unfortunate that one of these is `dxvk_` and the other is `dxvk-`. I've kept the naming from the Fedora packaging for now, but I'd be very tempted to change it to `dxvk_d3d9.pc`, `dxvk_glfw_dxgi.pc` and so on, to be consistent with the library names and avoid having two subtly different name-prefixes.

Reply:
It is unfortunate that Meson's `pkgconfig` module doesn't support setting `Provides:`. That would neatly resolve the issue, I think.